### ISSUE TITLE
Clicking on the "Get Started" button in an invitation to join "Shop Manager" group results in page with broken form.

### ISSUE THREAD

**Issue by noodle-t** [reactioncommerce/reaction#4892]
Issue tracker is **ONLY** used for reporting bugs. NO NEW FEATURE REQUESTS ACCEPTED. For new features, add an issue in our [Request Feature Request](https://github.com/reactioncommerce/reaction-feature-requests) repository.

Are you looking for help with getting started on Reaction? Please visit our [Reaction documentation](https://docs.reactioncommerce.com/reaction-docs/master/getting-started-developing-with-reaction).

## Prerequisites
* [x] Are you running the latest version?
* [x] Are you able to consistently reproduce the issue?
* [x] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
Providing a clear description of the issue will enable a developer to quickly reproduce and understand the issue. Please provide a detailed description including images, animation and/or videos. For client issues, include browser console errors, for server issues please add terminal output and stack traces.

## Steps to Reproduce
Please provide starting context, i.e. logged in as a user, configure a particular payment method.
1. On a new installation (Reproduced on both Windows 10 and Debian) Version 2.0.0-rc.7
2. Configure email to properly send email.
3. Send invitation email to join Shop Manager group
4. Go to email, and click button.
5. See blank page on Reaction site.
6. Looking at devtools console, I see:
Uncaught Error: Can't render undefined
    at checkRenderContent (blaze.js?hash=adc528…77eefb5def6bf1:2285)
    at contentAsFunc (blaze.js?hash=adc528…77eefb5def6bf1:2328)
    at Object.Blaze.renderWithData (blaze.js?hash=adc528…77eefb5def6bf1:2401)
    at Object.Accounts.onEnrollmentLink [as enroll-account] (updatePassword.js:16)
    at Meteor.startup (accounts_client.js:816)
    at maybeReady (meteor.js?hash=0504f…44eb6f53161cb63:927)
    at HTMLDocument.loadingCompleted (meteor.js?hash=0504f…44eb6f53161cb63:939)

Looking at code for Accounts.onEnrollmentLink I see:
Accounts.onEnrollmentLink((token, done) => {
  Blaze.renderWithData(Template.loginFormUpdatePasswordOverlay, {
    token,
    callback: done,
    isOpen: true,
    type: "setPassword"
  }, $("body").get(0));

Also in the code, and referencing earlier versions of the file /imports/plugins/core/accounts/client/templates/updatePassword/updatePassword.html
I see that this template was removed and replaced with loginFormUpdatePassword by user impactmass as described by this commit:
https://github.com/reactioncommerce/reaction/commit/9f8cc50f821fab7e1a419392878f9f5e80b79d4e#diff-13bf335625fad1b9d18abcf77e861a7d

I attempted to change the code in Accounts.onEnrollmentLink to use Template.loginFormUpdatePassword. This created a link in the email with no token, which caused the page to load without a failure, but also without a form.

At this point I figured I would report the issue and let someone who knows the code take a look at it.

## Possible Solution
Replace the use of loginFormUpdatePasswordOverlay with something that works since this template has apparently been removed. 

### Versions
(run `reaction -v` from your reaction directory)
Node: 9.11.1
NPM: 6.5.0
Meteor Node: 8.11.4
Meteor NPM: 6.4.1
Reaction CLI: 0.29.0
Reaction: 2.0.0-rc.7
Reaction branch: master
Docker: 17.09.0-ce

**Comment by NileThayne** [comment]
On a related issue, I find that if I go through the reset password process, the link brings me back to a "Page Not Found" error. By default the guest group does not get password reset privileges. Shouldn't that be on by default so that the password reset form will work? I imagine others may get hung up on this.

**Comment by zsimoes** [comment]
Same problem here, but the enrollment link takes me to a blank page. 
My stack trace is different:
```
Error: Can't render undefined 

checkRenderContent http://localhost:3000/packages/blaze.js:2285:11
contentAsFunc http://localhost:3000/packages/blaze.js:2328:3
Blaze.renderWithData http://localhost:3000/packages/blaze.js:2401:49
updatePassword.js/< updatePassword.js:16:2
defaultSuccessHandler/< accounts_client.js:816:39
maybeReady http://localhost:3000/packages/meteor.js:927:6
loadingCompleted http://localhost:3000/packages/meteor.js:939:5
```

**Comment by impactmass** [comment]
@NileThayne thanks for reporting. This issue will be fixed in #4919. Also, the reset password bug will be fixed in #4911
